### PR TITLE
LPD-26643 Address autoselect during checkout not working for account defaults and reorder

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-26643

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

File: modules/test/playwright/pages/commerce/commerce-checkout-web/checkoutPage.ts [hunk 1]
```diff
@@ -18,13 +18,15 @@ export class CheckoutPage {
 	readonly configurationMenuItem: Locator;
 	readonly continueButton: Locator;
 	readonly countryInput: Locator;
+	readonly buttonGoToOrderDetails: Locator;
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerce-checkout-web/checkoutPage.ts:21]: goToOrderDetailsButton

File: modules/test/playwright/pages/commerce/commerce-checkout-web/checkoutPage.ts [hunk 2]
```diff
@@ -18,13 +18,15 @@ export class CheckoutPage {
...
 	readonly layoutsPage: CommerceLayoutsPage;
 	readonly nameInput: Locator;
 	readonly optionsButton: Locator;
 	readonly orderSuccessMessage: Locator;
 	readonly page: Page;
 	readonly phoneNumberInput: Locator;
 	readonly regionInput: Locator;
+	readonly selectShippingAddress: Locator;
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerce-checkout-web/checkoutPage.ts:29]: shippingAddressSelect

File: modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts [hunk 3]
```diff
@@ -8,20 +8,34 @@ import {Locator, Page} from '@playwright/test';
...
 		);
 		this.headerDetailsTitle = page.getByTestId('headerDetailsTitle');
 		this.page = page;
+		this.reorderButton = page.getByRole('button', {
+			exact: true,
+			name: 'Reorder',
+		});
+	}
+
+	async reorderProduct() {
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts:37]: `reorder()`

File: modules/test/playwright/pages/users-admin-web/EditUserPage.ts [hunk 4]
```diff
@@ -95,14 +97,28 @@ export class EditUserPage {
...
+				.getByRole('link', {exact: true, name: siteName});
+		};
 		this.webDAVPasswordLabel = page.locator(
 			'#_com_liferay_users_admin_web_portlet_UsersAdminPortlet_webDAVPassword'
 		);
 		this.confirmButton = this.passwordConfirmationFrame.getByRole(
 			'button',
 			{name: 'Confirm'}
 		);
+		this.selectSiteForMembership = page.getByLabel('Select Sites');
```
Comment (alessiorendina) [inline modules/test/playwright/pages/users-admin-web/EditUserPage.ts:112]: Use a better locator, try with `getByRole`

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@monaco-fabio please check my comments, thanks!

**Comment by monaco-fabio** [comment]
ci:stop

**Comment by monaco-fabio** [comment]
#5169

**Pull request closed without merging**